src/uct/ib/base/ib_device.c

```diff
@@ -435,9 +451,10 @@ void uct_ib_handle_async_event(uct_ib_device_t *dev, uct_ib_async_event_t *event
         level = UCS_LOG_LEVEL_DEBUG;
         break;
     case IBV_EVENT_DEVICE_FATAL:
+        uct_ib_device_async_event_dispatch(dev, event);
```

Comment:
we should probably have another function here, like uct_ib_device_async_event_dispatch_all()
because all what uct_ib_device_async_event_dispatch is doing now is if between single dispatch and dispatch-all
and then can probably keep existing uct_ib_device_async_event_dispatch() as -is